### PR TITLE
EN-70078: jenkinsfile templating

### PR DESCRIPTION
**Pull request by sarah-salinas**
This commit tests out the jenkinsfile templating PR https://github.com/socrata/pipelines/pull/167 by implementing it for this service in a testable way.

### CODE REVIEW

**Comment by sarah-salinas** [comment]
Testing documented here: https://github.com/socrata/pipelines/pull/167#issuecomment-2489514342

**Review by rjmac — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -1,211 +1,16 @@
 @Library('socrata-pipeline-library')
 
-import com.socrata.ReleaseMetadataService
-def rmsSupportedEnvironment = com.socrata.ReleaseMetadataService.SupportedEnvironment
+Map pipelineParams = [
+  projectName: 'query-coordinator',
```
Comment (ds2643) [inline Jenkinsfile:4]: I suggest sorting these keys to make finding keys easier.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,211 +1,16 @@
...
+  language: 'scala',
+  defaultBuildWorker: 'build-worker-pg13',
+  numberOfBuildsToKeep: 50,
+  teamsChannelWebhookId: 'WORKFLOW_IQ',
+  projectWorkingDirectory: 'query-coordinator',
+  languageVersion: '2.12',
+  scalaSubprojectName: "queryCoordinator",
+  scalaSrcJar: "query-coordinator/target/query-coordinator-assembly.jar",
```
Comment (ds2643) [inline Jenkinsfile:12]: No interpolation is necessary here, so you can change the double quotes to single. In other words, the value associated with `scalaSrcJar` should be of type String, not GString.

File: Jenkinsfile (hunk 1 above)
Comment (ds2643) [reply Jenkinsfile:4]: Yes.

Comment (sarah-salinas) [reply Jenkinsfile]: Done

File: Jenkinsfile (hunk 2 above)
Comment (sarah-salinas) [reply Jenkinsfile]: Done

**Pull request merged**